scripts/mainPage.js

```diff
@@ -0,0 +1,146 @@
+const marqueeInnerText = document.getElementById("marquee-text");
+const input = document.getElementById("searchLine");
+const searchBtn = document.getElementById("searchBtn");
+const loadingSpinner = document.getElementById("loadingSpinner");
+let baseUrl = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/`;
+
+searchBtn.addEventListener("click", function (event) {
```

Comment:
better to add the event listener after the deleration on getDataFromAPI. don't trust hoisting.